data/json/recipes/recipe_food.json

```diff
@@ -7693,6 +7693,18 @@
   ]
 },{
   "type" : "recipe",
+  "result": "vodka",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_DRINKS",
+  "difficulty": 0,
+  "time": 20,
+  "reversible": false,
+  "autolearn": true,
+  "components": [
+    [ ["chem_ethanol", 1] ],
```

Comment:
Should be like, 7 or so. Match the volume of input and output.

Also, test the results. I'm pretty sure this recipe will produce 14 vodkas, not 1.